frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx

```diff
@@ -1,79 +1,188 @@
+            if (selectedOption === null) {
+                setPeopleDropdown([]);
+            } else {
+                setPeopleDropdown(selectedOption);
+            }
+        },
+        [],
+    );
+
+    const handleCategoriesMultiDropdownChange = useCallback(
+        (selectedOption: MultiValue<DataType> | SingleValue<DataType>) => {
+            if (selectedOption === null) {
+                setCategoriesDropdown([]);
+            } else {
+                setCategoriesDropdown(selectedOption);
+            }
+        },
+        [],
+    );
+
+    const handleSliderChange = useCallback((range: RangeLimits): void => {
+        setCurrentRange(range);
+
+        const newFilteredData = mockSliderData.filter(
+            (item) => item.amount >= range.min && item.amount <= range.max,
+        );
+        setFilteredData(newFilteredData);
+    }, []);
+
+    const hangleReset = useCallback((): void => {
+        setPeopleDropdown([]);
+        setCategoriesDropdown([]);
+        setFilteredData(mockSliderData);
+        setCurrentRange(rangeLimits);
+    }, [rangeLimits]);
+
     return (
         <div className={styles.app}>
-            <div className={styles.header}>
-                <div className={styles.headerContainer}>
-                    <div className={styles.logo}>
-                        <div className={styles.logoIcon}></div>
-                        <span className={styles.logoText}>SmartSpend</span>
-                    </div>
-                    <div className={styles.tabs}>Add tabs here</div>
-                    <div className={styles.profile}>
-                        {/*There should be an img*/}
-                        <span className={styles.userName}>Ann Rose</span>
-                    </div>
-                </div>
-            </div>
             <div className={styles.body}>
                 <div className={styles.bodyContainer}>
                     <div className={styles.buttonsDate}>
-                        <Button
-                            variant={ButtonVariant.PRIMARY}
-                            size={ButtonSize.MEDIUM}
-                        >
-                            <span>+</span>
-                            <span>Add transaction</span>
-                        </Button>
                         <div className={styles.buttonsContainer}>
                             <Button
-                                variant={ButtonVariant.SECONDARY}
-                                size={ButtonSize.MEDIUM}
-                            >
-                                Future
-                            </Button>
-                            <Button
-                                variant={ButtonVariant.SECONDARY}
+                                variant={ButtonVariant.PRIMARY}
                                 size={ButtonSize.MEDIUM}
                             >
-                                Import
+                                <span>+</span>
```

Comment:
May have used the plus icon from the library?
![Code_1BBU9SSKho](https://user-images.githubusercontent.com/16450566/229155478-afa69ba7-28a6-4e65-807b-3e81751bf771.png)

Reply:
fixed